src/xunit.netcore.extensions/xunit.netcore.extensions.csproj

```diff
@@ -10,6 +10,8 @@
     <RootNamespace>Xunit.NetCore.Extensions</RootNamespace>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
```

Comment:
Yeah, this shouldn't be necessary anymore. It looks like the latest builds are built against a portable profile, as of this change:

https://github.com/xunit/xunit/commit/4089905f0e7ba13160f6048b32b3a3988eb203dc#diff-7ffcd805a457da27f45456f40843a2d9R16
(Warning: Might crash your browser)